src/app/centers/centers.module.ts

```diff
@@ -12,7 +12,7 @@ import { CentersService } from './centers.service';
 
 @NgModule({
   imports: [
-    CoreModule,
```

Comment:
SharedModule should be imported for all components, and CoreModule should never be imported apart from in the main app module.